marketpulse/auth/backend.py

```diff
@@ -50,6 +50,11 @@ def __init__(self, *args, **kwargs):
 
         self.mozillians_client = MozilliansClientLegacy(self.api_url, self.api_key, self.app_name)
 
+    def _normalize_email(self, email):
+        email_name, domain = email.strip().rsplit('@', 1)
+        email = '@'.join([email_name, domain.lower()])
```

Comment:
RFC for email says that ``email_name`` is case sensitive. This is also the way django internally normalizes emails.